### ISSUE TITLE
[Issue] Turkish (or any other keyboard type) Q needs a fix in the cheatsheet.

### ISSUE THREAD

**Issue by enessmr** [end-4/dots-hyprland#958]
- I have read the Usage and Troubleshooting pages of the wiki: https://end-4.github.io/dots-hyprland-wiki/en/i-i/04troubleshooting/
- I have made sure that both my config and system packages are up to date
- Linux distro: 
 Arch Linux
---
## The issue

You set the Slash in the thing, so i had to use another key to work in TRQ.
So.. Super+shift,/ does nothing at all
Super+shift,. does nothing at all
Super+shift,i works

<details>
<summary>Logs</summary>

```plain
no log cause it's a keybind issue
```

</details>

**Comment by os-guy** [comment]
That's true. I also have TRQ Keyboard Layout. '/' key or '.' key doesn't work.

**Comment by Pharmaracist** [comment]
you can change this in keybinds.conf
